luigi/tools/range.py

```diff
@@ -536,7 +536,7 @@ def most_common(items):
     for i in items:
         counts.setdefault(i, 0)
         counts[i] += 1
-    return max(six.iteritems(counts), key=operator.itemgetter(1))
+    return max(counts.items(), key=operator.itemgetter(1))
```

Comment:
https://github.com/spotify/luigi/pull/2834/files#diff-b81d2935d10c6b5ffb3685b1f91fe1d3R533 looks a bit weird now.

Reply:
@honnix 
link seems to be broken
could you please name the file?

Reply:
Yeah a new commit broke it. The same file same method. `Wanted functionality from Counters (new in Python 2.7).`

Reply:
@honnix done